contrib/drivers/pgsql/pgsql_convert.go

```diff
@@ -128,7 +128,7 @@ func (d *Driver) ConvertValueForLocal(ctx context.Context, fieldType string, fie
 		if err := result.Scan(fieldValue); err != nil {
```

Comment:
I found it here should be `result.Scan(&fieldValue)`.

For example:
```go
package main

import (
	"fmt"

	"github.com/gogf/gf/v2/util/gconv"
	"github.com/lib/pq"
)

func main() {
	var s pq.StringArray
	err := gconv.Scan([]string{"a", "b"}, &s)
	fmt.Println(err)
	fmt.Println(s)
}
```

Reply:
I don't think so.  fieldValue is of type []byte. The pg.StringArray Scan method only accepts []byte, string, or nil, and using &fieldValue as *[]byte would result in a conversion error.